pyEPR/calcs/back_box_numeric.py

```diff
@@ -267,7 +267,7 @@ def PT_on_vector(original_vector, original_basis, pertub, energy0, evalue):
 
         def closest_state_to(s):
             def distance(s2):
-                return (s.dag() * s2[1]).norm()
+                return np.abs((s.dag() * s2[1]))
```

Comment:
Great!

Have you checked if this works with the new version? 

And ideally if it is backwards compatible 

There is also this possibility I think if not mistaken 

return np.abs(np.vdot(s, s2[1]))

Reply:
Just tested it with a fresh install with python3.10 and latest numpy. Ran into no problems. Numpy was a red herring: it turns out that that was not the problem after all. I don't know exactly was, but the pull request fixes it.

Reply:
Thank you Garry! Going to merge